terraform/application/application.tf

```diff
@@ -25,6 +25,7 @@ module "application_configuration" {
   }
   secret_variables = {
     DATABASE_URL = module.postgres.url
+    REDIS_CACHE_URL = var.deploy_redis_cache ? module.redis-cache[0].url : ""
```

Comment:
should this be ` module.redis-cache.url` 🤔 rather than `module.redis-cache[0].url`?

Reply:
I thought so too, but I think because I'm using `count` to determine if it should spin up a Redis instance this ends up being a collection. I'll make sure to double check the env variable when I deploy it before I merge this down though.